deployment/terraform/resources/variables.tf

```diff
@@ -34,6 +34,16 @@ variable "pctasks_server_replica_count" {
   default = 1
 }
 
+variable "aks_task_group_label" {
```

Comment:
This is unused... Delete or figure out what it was going to be used for (maybe for labels on the Kubernetes pods, but why would that need to be a variable?)